docs/KUBERNETES.md

```diff
@@ -187,6 +187,29 @@ Alternatively, update the DaemonSet configuration directly by using the followin
 kubectl patch daemonset -n logdna-agent logdna-agent --type json -p '[{"op":"add","path":"/spec/template/spec/containers/0/securityContext/runAsUser","value":5000},{"op":"add","path":"/spec/template/spec/containers/0/securityContext/runAsGroup","value":5000}]'
 ```
 
+### Enabling file offset tracking across restarts
+
+To avoid possible duplication or skipping of log messages during agent restart or upgrade, the agent stores its current file offsets on the host node's filesystem, using a hostPath volume.
```

Comment:
Use host or node but not both.

```suggestion
To avoid possible duplication or skipping of log messages during agent restart or upgrade, the agent stores its current file offsets on the host's filesystem, using a `hostPath` volume.
```